### ISSUE TITLE
Add list_distributions example to repo readme

### ISSUE THREAD

**Issue by nlinton** [epiverse-trace/epiparameter#245]
As a user, if I want to use the package I first need an accessible way to see if the disease I am working on has any entry, how many entries there are for that disease, etc. (I.e. see what my options are). It took some digging to find the `list_distributions` function that had what I was looking for. It would be great if you added the `list_distributions` example to the repo readme so it is easier for users to find.

**Comment by nlinton** [comment]
It would also be immensely helpful for `list_distributions` to output pathogen as the second column.

**Comment by joshwlambert** [comment]
@nlinton Thank you for your suggestions. I will work on improving the README and adding the pathogen column. 

Also just to add, the `list_distributions()` function will likely be renamed to `parameter_tbl()` as the current name is a little misleading about the return type.

**Comment by nlinton** [comment]
Sounds good, and thanks for the heads up!

**Comment by joshwlambert** [comment]
@nlinton apologies for the slow response and getting these changes merged into the `main` branch. There is a pull request (#321) open now which include the changes you've requested and will be merged later today. The issue will close once the PR is merged. 

If the changes are not exactly what you were after, or you'd like to make another request please feel free to reopen this issue or open a new issue.

Thanks for the input to improve the package!
